pkg/observable/channel/observable_test.go

```diff
@@ -16,70 +16,70 @@ import (
 )
 
 const (
-	productionDelay        = 2 * time.Millisecond
-	notifyTimeout          = productionDelay * 5
-	cancelUnsubscribeDelay = notifyTimeout * 2
+	publishDelay           = 100 * time.Microsecond
+	notifyTimeout          = publishDelay * 20
+	cancelUnsubscribeDelay = publishDelay * 2
 )
 
 func TestChannelObservable_NotifyObservers(t *testing.T) {
 	type test struct {
 		name            string
-		producer        chan *int
+		publishCh       chan int
 		inputs          []int
 		expectedOutputs []int
 		setupFn         func(t test)
 	}
 
 	inputs := []int{123, 456, 789}
-	// NB: see INCOMPLETE comment below
-	// fullBlockingProducer := make(chan *int)
-	// fullBufferedProducer := make(chan *int, 1)
+	// NB: see TODO_INCOMPLETE comment below
+	//fullBlockingPublisher := make(chan *int)
+	//fullBufferedPublisher := make(chan *int, 1)
 
 	tests := []test{
 		{
-			name:            "nil producer",
-			producer:        nil,
+			name:            "nil publisher",
+			publishCh:       nil,
 			inputs:          inputs,
 			expectedOutputs: inputs,
 		},
 		{
-			name:            "empty non-buffered producer",
-			producer:        make(chan *int),
+			name:            "empty non-buffered publisher",
+			publishCh:       make(chan int),
 			inputs:          inputs,
 			expectedOutputs: inputs,
 		},
 		{
-			name:            "empty buffered len 1 producer",
-			producer:        make(chan *int, 1),
+			name:            "empty buffered len 1 publisher",
+			publishCh:       make(chan int, 1),
 			inputs:          inputs,
 			expectedOutputs: inputs,
 		},
-		// INCOMPLETE: producer channels which are full are proving harder to test
+		// INCOMPLETE: publisher channels which are full are proving harder to test
 		// robustly (no flakiness); perhaps it has to do with the lack of some
 		// kind of guarantee about the receiver order on the consumer side.
 		//
 		// The following scenarios should generally pass but are flaky:
```

Comment:
Can you insert links to the github images from https://github.com/pokt-network/poktroll/pull/31#discussion_r1364559841

Reply:
Created https://github.com/pokt-network/poktroll/issues/81 and added `TODO_INCOMPLETE(#81)`.